acceptance/localcluster/docker.go

```diff
@@ -146,6 +146,12 @@ func maybePanic(err error) {
 // Remove removes the container from docker. It is an error to remove a running
 // container.
 func (c *Container) Remove() error {
+	if os.Getenv("CIRCLE_ARTIFACTS") != "" {
+		// HACK: Removal of docker containers fails on circleci with the error:
```

Comment:
Nope. I thought it was related to docker version, but circleci upgraded to docker 1.6 recently and is still experiencing the error. It's not a big deal for us, though as they tear down docker at the end of each run causing all containers to be implicitly removed.